netbox/netbox/urls.py

```diff
@@ -68,6 +71,41 @@
 
 ]
 
+# Plugins
+plugin_patterns = []
+plugin_api_patterns = []
+for app in apps.get_app_configs():
```

Comment:
Should we move this logic out of the root URLs file? Maybe into `extras/plugins/urls.py`?

Reply:
Yeah I like it, will do.